docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -116,6 +117,17 @@ const copyTextToClipboard = (text: string) => {
   modal.removeChild(textArea);
 };
 
+const parseServerRootPath = (docServiceRoute: string) => {
+  const plainRoute = docServiceRoute.replace('/*', '');
```

Comment:
What do you think of skipping unless the route ends with `/*`?
I'm thinking of regex cases where the path may look like: `a/**/b`

Reply:
In backend we configured it in a way that it only works for `PREFIX` paths. If it is something else, `docServiceRoute` is going to be empty string. But it makes sense to just have the logic here as a safe-guard.

Reply:
Updated logic to check route type.